### PR TITLE
Clean up contact margin warnings

### PR DESCRIPTION
**Pull request by Levi-Armstrong**

### CODE REVIEW

File: trajopt/src/collision_terms.cpp [hunk 1]
```diff
@@ -917,7 +917,8 @@ SingleTimestepCollisionEvaluator::SingleTimestepCollisionEvaluator(
 
   contact_manager_ = env_->getDiscreteContactManager();
   contact_manager_->setActiveCollisionObjects(adjacency_map_->getActiveLinkNames());
-  contact_manager_->setContactDistanceThreshold(safety_margin_data_->getMaxSafetyMargin() + safety_margin_buffer_);
+  /** @todo Should remove trajopt safety margin data structure and use the one from tesseract */
```
Comment (mpowelson) [inline trajopt/src/collision_terms.cpp:920]: I believe I did this in #219 for trajopt_ifopt.

**Review by mpowelson — APPROVED** [review]
Thanks for doing this.

File: trajopt/src/collision_terms.cpp (hunk 1 above)
Comment (Levi-Armstrong) [reply trajopt/src/collision_terms.cpp:920]: Great

> Thanks for doing this.

No problem.

**Comment by Levi-Armstrong** [comment]
These changes are included in PR #224

**Pull request closed without merging**